multicluster/cmd/check.go

```diff
@@ -107,6 +109,7 @@ non-zero exit code.`,
 	}
 	cmd.Flags().StringVarP(&options.output, "output", "o", options.output, "Output format. One of: table, json, short")
 	cmd.Flags().DurationVar(&options.wait, "wait", options.wait, "Maximum allowed time for all tests to pass")
+	cmd.Flags().DurationVar(&options.timeout, "timeout", options.timeout, "timeout duration between running checks")
```

Comment:
All the other help text starts with an uppercase letter, so let's uppercase this as well.
```suggestion
	cmd.Flags().DurationVar(&options.timeout, "timeout", options.timeout, "Timeout duration between running checks")
```

Reply:
Also, I'm not sure if I love the wording "between running checks", that kind of implies that we wait for this period of time after running a check prior to starting the next one...

Reply:
this is specifically a timeout for calls to the kubernetes API, so we probably want to be clear about that in the flag description.

Reply:
@hawkw @adleong I had my suspicions about the description too! I'll push the changes, although I still don't completely understand what this timeout duration exactly mean.

Reply:
How about: "Timeout for calls to the Kubernetes API"?

Reply:
```suggestion
	cmd.Flags().DurationVar(&options.timeout, "timeout", options.timeout, "Timeout for calls to the Kubernetes API")
```